### ISSUE TITLE
monster priority

### ISSUE THREAD

**Issue by Nasteff** [EasyFarm/EasyFarm#186]
is there a way to make it attack mobs that are already attacking you first?  Im having an issue where i get bound and my target is too far away for me to hit, meanwhile several monsters already hitting me kill me because im not getting backup from trusts unless i hit the monster.

**Comment by Mykezero** [comment]
Yeah, that's definitely not an ideal situation. This will require a new way of handling targeting when the player comes bound but there are enemies that could be attacked.
